src/commands.rs

```diff
@@ -3968,6 +3968,7 @@ fn cmd_git_remote_list(
     Ok(())
 }
 
+#[tracing::instrument(skip(ui, command))]
```

Comment:
Is it useful to log args here? The user presumably knows/can see what they passed.

Reply:
I was going with the approach of logging anything that had a debug impl. Having the args running through into verbose logs might also help if a user is sharing only their logs for output and not their command.